db/migrations/0003_add_frames_table.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE frames (
```

Comment:
do we need to have next? Id like frames to be immutable and if we have next it wont be